### PR TITLE
chore(external docs): Add `statsd` sink metadata

### PR DESCRIPTION
**Pull request by binarylogic**
Signed-off-by: binarylogic <bjohnson@binarylogic.com>

### CODE REVIEW

File: .meta/sinks/statsd.toml.erb [hunk 1]
```diff
@@ -1,41 +0,0 @@
...
-description = "The UDP socket address to send stats to."
-
-[sinks.statsd.options.namespace]
-type = "string"
-common = true
-examples = ["service"]
-required = false
-description = "A prefix that will be added to all metric names."
-
-[[sinks.statsd.examples]]
```
Comment (jamtur01) [inline .meta/sinks/statsd.toml.erb:35]: We don't use examples anymore?

File: .meta/sinks/statsd.toml.erb [hunk 2]
```diff
@@ -1,41 +0,0 @@
-[sinks.statsd]
-title = "Statsd"
-noun = "Statsd"
-beta = false
-common = false
-delivery_guarantee = "best_effort"
-<%= render("_partials/descriptions/_statsd.toml") %>
-egress_method = "streaming"
-features = [
```
Comment (jamtur01) [inline .meta/sinks/statsd.toml.erb:9]: We don't use features anymore?

**Review by jamtur01 — COMMENTED** [review]
Couple of quick questions.

File: .meta/sinks/statsd.toml.erb (hunk 1 above)
Comment (binarylogic) [reply .meta/sinks/statsd.toml.erb:35]: Yeah, this is what I need help with as described in #4303 😄 .

File: .meta/sinks/statsd.toml.erb (hunk 2 above)
Comment (binarylogic) [reply .meta/sinks/statsd.toml.erb:9]: No, I am dropping those, they were for SEO purposes and we don't need them.

**Comment by binarylogic** [comment]
cc @juchiast can you make sure the options are correct given https://github.com/timberio/vector/pull/3557?

**Comment by juchiast** [comment]
@binarylogic this commit https://github.com/timberio/vector/commit/5938e57991143161d7f89a22920497f01f6556b2 override the changes that you made.

**Pull request merged**